dev/cocoon_try_builders.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
For the test I was thinking something like a dart script that receives a the path of the json file as a flag and validates it has the correct content and the file is parseable.

Reply:
I have added a separate dart file to validate the json contents: `validate_json.dart`, and included it in the README.

Reply:
> I have added a separate dart file to validate the json contents: `validate_json.dart`, and included it in the README.

Great, maybe not in this PR but can we run the validation in presubmit?

Reply:
I will try adding that in a separate PR.